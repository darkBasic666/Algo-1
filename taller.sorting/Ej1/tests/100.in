100
58962 66848 48461 25227 5943 69823 61483 88335 78201 76929 54487 16874 97153 81491 98392 41155 32855 92764 12969 9072 41224 32135 30540 78323 49627 31553 37006 82005 15563 71968 92400 70348 56201 80359 96893 40212 4411 63597 5245 59776 26667 77202 97399 9489 14770 78596 400 67254 63913 54436 94677 90193 43189 70060 66562 2168 66945 13536 58429 57507 98507 93948 63497 39430 1414 713 77238 49792 3910 46143 49227 11034 68738 83694 94757 95927 45571 23844 25025 44754 6619 23559 80771 72005 78385 82183 71224 42498 16283 70791 69125 49264 52547 64829 12413 29672 84490 31130 3352 30640 