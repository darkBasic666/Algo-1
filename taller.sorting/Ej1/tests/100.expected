4.5e-05secs.
100
400 713 1414 2168 3352 3910 4411 5245 5943 6619 9072 9489 11034 12413 12969 13536 14770 15563 16283 16874 23559 23844 25025 25227 26667 29672 30540 30640 31130 31553 32135 32855 37006 39430 40212 41155 41224 42498 43189 44754 45571 46143 48461 49227 49264 49627 49792 52547 54436 54487 56201 57507 58429 58962 59776 61483 63497 63597 63913 64829 66562 66848 66945 67254 68738 69125 69823 70060 70348 70791 71224 71968 72005 76929 77202 77238 78201 78323 78385 78596 80359 80771 81491 82005 82183 83694 84490 88335 90193 92400 92764 93948 94677 94757 95927 96893 97153 97399 98392 98507 